### ISSUE TITLE
Wrong OS checking in Vulnerability detector causes "duplicated" alerts

### ISSUE THREAD

**Issue by jesusgn90** [wazuh/wazuh#720]
Hello team, just detected duplicated alerts at the same scan. Read below for details:

Alert 1:

```
    @timestamp": "2018-06-05T13:34:25.838Z",
    ...
      "vulnerability": {
        "severity": "Critical",
        "state": "Fixed",
        "cve": "CVE-2017-7810",
        "title": "RHSA-2017:2831: firefox security update (Critical)",
        "updated": "2017-09-28",
        "package": {
          "patch": "RHSA-2017:2831-01",
          "condition": "less than 0:52.4.0-1.el6_9",
          "version": "52.2.0-2.el7.centos",
          "cvss3": "8.8/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
          "name": "firefox"
        },
        "reference": "https://access.redhat.com/security/cve/CVE-2017-7810",
        "published": "2017-09-28"
      }
```
Alert 2:

```
    "@timestamp": "2018-06-05T13:34:25.840Z",
    ...
     "vulnerability": {
        "severity": "Critical",
        "state": "Fixed",
        "title": "RHSA-2017:2831: firefox security update (Critical)",
        "updated": "2017-09-28",
        "cve": "CVE-2017-7810",
        "package": {
          "patch": "RHSA-2017:2831-01",
          "condition": "less than 0:52.4.0-1.el7_4",
          "version": "52.2.0-2.el7.centos",
          "cvss3": "8.8/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
          "name": "firefox"
        },
        "reference": "https://access.redhat.com/security/cve/CVE-2017-7810",
        "published": "2017-09-28"
      }
```

The condition is the problem here:

```
          "condition": "less than 0:52.4.0-1.el6_9",
```

and 

```
          "condition": "less than 0:52.4.0-1.el7_4",
```

This means it's checking for CentOS 6 and 7. I've not configured for CentOS 6 so it's wrong.

Regards,
Jesús

**Comment by chemamartinez** [comment]
Taking into account here: https://github.com/wazuh/wazuh/issues/1887

**Comment by crolopez** [comment]
This was fixed in Wazuh 3.8.0.
